welcome_wizard/banner.py

```diff
@@ -0,0 +1,32 @@
+"""Home page banner for Welcome Wizard."""
+from typing import Optional
+
+from django.urls import reverse
+from django.utils.html import format_html
+
+from nautobot.extras.choices import BannerClassChoices
+from nautobot.extras.plugins import PluginBanner
+
+
+def banner(context, *args, **kwargs) -> Optional[PluginBanner]:
+    """
+    Construct the Welcome Wizard Banner on the Home Page.
+
+    These conditions must be met:
+    - If User has permissions to the Welcome Wizard Dashboard (merlin)
+    - On the Nautobot Home Page
+    """
+    content = format_html(
+        "<a href={}>The Nautobot Welcome Wizard can help you get started with Nautobot!</a>",
+        reverse("plugins:welcome_wizard:dashboard"),
+    )
```

Comment:
It'd be very slightly more efficient to move this inside the `if` block below so that it doesn't get constructed unnecessarily - not blocking.